fiftyone/utils/map.py

```diff
@@ -56,10 +63,12 @@ def update_samples(
     sample_collection,
```

Comment:
Same as above method, either remove the types or make them correct and simplified.